keyboards/sofle/keymaps/default/config.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+/* The way how "handedness" is decided (which half is which),
+see https://docs.qmk.fm/#/feature_split_keyboard?id=setting-handedness
+for more options.
+*/
+#define MASTER_LEFT
```

Comment:
Not actually needed.  This is the default behavior if no other handedness defines are present. 

```suggestion
```